clippy_tests/examples/range.rs

```diff
@@ -29,7 +21,4 @@ fn main() {
     let v2 = vec![4,5];
     let _x = v1.iter().zip(0..v1.len());
     let _y = v1.iter().zip(0..v2.len()); // No error
-
-    // check const eval
-    let _ = v1.iter().step_by(2/3);
```

Comment:
Sorry, that wasn't very clear... I meant to leave all the calls to step_by in the tests. At least those that compile.